cherab/tools/observers/group/sightline.py

```diff
@@ -18,10 +18,11 @@
 # under the Licence.
 
 from cherab.tools.observers.spectroscopy import SpectroscopicSightLine
+from .spectroscopic import SpectroscopicObserver0DGroup
 from .base import Observer0DGroup
 
 
-class SightLineGroup(Observer0DGroup):
+class SightLineGroup(SpectroscopicObserver0DGroup):
```

Comment:
Similar to `FibreOpticGroup`, this class should inherit from `Observer0DGroup` and use `SightLine` fomr Raysect instead of `SpectroscopicSightLine` from Cherab.